docs/installguide/install_all.rst

```diff
@@ -0,0 +1,307 @@
+with ``dpkg -i  ka-lite_0.x.x-0ubuntu1_all.deb``.
+
+The file may be named as if it was intended for Ubuntu but works just as well
+for any other Debian-based systems like Debian, Raspberry Pi, Linux Mint etc.
+
+Configuration after installing / updating
+_________________________________________
+
+Every time you install or update kalite, you should (re)run ``kalite manage setup``
+to setup the database and download assessment items (video descriptions,
+exercises etc.).
+
+
+.. _linux-pypi-install:
+Linux: Installing as a PyPI package
+===================================
+
+To install KA Lite from a file, go to our `PyPi page <https://pypi.python.org/pypi/ka-lite-static>`_ and download a .tar.gz or .zip. You can then install the file from a system by unpacking it and running *python setup.py install* or *pip install ka-lite-static-X.Y.Z.tar.gz*.
+
+Alternatively, install with *pip* using the instructions below:
+
+.. note:: Type the following commands in a terminal.
+
+#. Install pip (Ubuntu: ``sudo apt-get install python-pip``)
+#. (Recommended; essential on slower platforms like Raspberry Pi) Install M2Crypto (``sudo apt-get install python-m2crypto``).
+#. Run ``sudo pip install ka-lite-static`` (bundled dependencies) or ``sudo pip install ka-lite`` (normal pip behaviour, dependencies installed to host system).
+#. Run ``kalite manage setup``.
+
+For more tips see :ref:`advanced-installation`.
+
+Configuration after installing / updating
+_________________________________________
+
+Every time you install or update kalite, you should (re)run ``kalite manage setup``
+to setup the database and download assessment items (video descriptions,
+exercises etc.).
+
+
+Uninstalling
+============
+
+You can remove KA Lite (when installed from pip or source distribution) with
+`pip uninstall ka-lite` or `pip uninstall ka-lite-static` (static version).
+
+For Ubuntu, use ``apt-get remove ka-lite``.
+
+
+Removing user data
+------------------
+
+Downloaded videos and database files are in `~/.kalite`. Navigate to the
+home directory of the user who ran the KA Lite server and remove that directory
+to potentially reclaim lots of hard drive space.
+
+
+
+Raspberry Pi
+============
+
+For a Raspberry Pi running a Debian system, you can install the Debian package.
+
+
+
+Raspberry Pi Wi-Fi
+------------------
+
+.. note:: Two Wi-Fi USB modules have been tested with KA Lite on the Raspberry Pi
+
+    * Raspberry Pi WiPi adaptor
+    * Edimax EW-7811Un
+
+In our tests, we found that the WiPi adaptor supported a higher number tablet connections.
+
+
+.. note:: The Raspberry Pi may crash if the USB adaptor is inserted or removed while the computer is switched on.
+
+    * Make sure to shutdown and remove the power from the Raspberry Pi.
+    * Afterwards, insert the wireless USB adaptor.
+    * Lastly, switch the Raspberry Pi on.
+
+#. Make sure the Raspberry Pi operating system is up-to-date.

```

Comment:
I fixed up lots of stuff... and lots of stuff I didn't fix. But it looks like I wrote everything... but that's the sphinx-docs -> docs rename.